docs/content/Getting-Started/user-quickstart-test.md

```diff
@@ -384,6 +399,11 @@ how to create, but not overwrite/update a synchronized resource
 <br>
 
 ## Tear it all down
+``` {.bash .hide-me}
+: not really
```

Comment:
I'm not following this line here. May I have some more explanations?

Reply:
For a contributor doing testing, it is sometimes counter-productive to have the test destroy the state it developed. I find it helpful to be able to do further investigations after the test completed.

Reply:
I have revised this PR to accomplish the goal in a non-hacky way.

Reply:
I see. Thank you!